### PR TITLE
lint aliases, with prettier too

### PR DESCRIPTION
**Pull request by BarryThePenguin**
Saw you had issues recently where a `lodash` package was in devDependencies and not dependencies.

Also added the prettier plugin for eslint. Hope it helps!

**Unresolved dependencies**

```
❯ yarn lint
yarn run v1.16.0
$ eslint .

/server/lib/orm/associations/has-many.js
  4:21  error  Unable to resolve path to module 'lodash.compact'  import/no-unresolved

/server/lib/orm/model.js
  8:21  error  Unable to resolve path to module 'lodash.compact'  import/no-unresolved

/server/lib/serializer.js
  11:21  error  Unable to resolve path to module 'lodash.compact'  import/no-unresolved

/server/lib/serializers/json-api-serializer.js
  6:21  error  Unable to resolve path to module 'lodash.compact'  import/no-unresolved

✖ 4 problems (4 errors, 0 warnings)
```

**`devDependencies` used inside `/lib`**

```
❯ yarn lint
yarn run v1.16.0
$ eslint .

/server/lib/trait.js
  1:1  error  'prettier' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies

✖ 1 problem (1 error, 0 warnings)
```

### CODE REVIEW

File: __tests__/internal/unit/serializers/rest-serializer-test.js [hunk 1]
```diff
@@ -1,9 +1,4 @@
-import {
-  _ormSchema as Schema,
-  _Db as Db,
-  Model,
-  RestSerializer
-} from "@lib";
+import { _ormSchema as Schema, _Db as Db, Model, RestSerializer } from "@lib";
```
Comment (BarryThePenguin) [inline __tests__/internal/unit/serializers/rest-serializer-test.js]: This change was made by `prettier` when running `yarn lint --fix`

**Comment by ryanto** [comment]
These changes look great!

It looks like travis didn't report any lint errors, in your PR you mentioned a prettier dependency lint error:

```
  1:1  error  'prettier' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
```

I'm wondering where this came from or if anything needs to be done to address it?

**Comment by BarryThePenguin** [comment]
Oh, they were examples of the errors it would report 😅

**Comment by samselikoff** [comment]
@BarryThePenguin but those aren't actual errors right now right? This PR is good?

**Comment by samselikoff** [comment]
(the error about `'prettier' should be listed in the project's dependencies` is confusing, I don't think that's right?)

**Comment by BarryThePenguin** [comment]
Yes, the PR should be good.

The errors I listed above are examples of the types of errors we will see with this linter configuration

Sorry for the confusion!

**Comment by samselikoff** [comment]
Awesome. Thank you!

**Pull request merged**